R/sv_to_custom_track.R

```diff
@@ -14,13 +14,13 @@
 #'
 #' @return Nothing.
 #'
-#' @import dplyr tidyr
+#' @import dplyr tidyr GAMBLR.helpers
```

Comment:
Should this function also import `GAMBLR.viz`?

Reply:
Added this now! Thanks for noticing!